macat/macat.go

```diff
@@ -430,9 +430,6 @@ func recvLoop(sock mangos.Socket, done chan struct{}) {
 		}
 		printMsg(msg)
 		msg.Free()
-		if sendInterval == 0 {
```

Comment:
This isn't quite right.  In particular, if this code is applied, then it breaks the one-shot semantic applied to other protocols.  For example, req/rep normally uses no argument to get a one-shot semantic, but with this change the req socket blocks ~forever.  Now nanocat is weird in this case, because nanocat seems not to get the reply from the request, or at least it doesn't bother to print it.